### PR TITLE
Fixed title and spacing

### PR DESCRIPTION
**Pull request by harshavardhanchintapatla**

### CODE REVIEW

File: fatplant/src/app/components/pages/datapages/protein-details-new/protein-details-new.component.scss [hunk 1]
```diff
@@ -89,6 +95,8 @@ a:visited {
   display: flex;
   justify-content: space-between;
   max-height: 50px;
+  margin-top: 1%;
```
Comment (Yongfang-Qin) [inline fatplant/src/app/components/pages/datapages/protein-details-new/protein-details-new.component.scss:98]: what's this for?

File: fatplant/src/app/components/pages/datapages/protein-details-new/protein-details-new.component.html [hunk 2]
```diff
@@ -52,14 +55,14 @@ <h2 *ngIf="homologs" class='title'>Homologs</h2>
                                     <h3 class='protein-subtitle'>Identification</h3>
                                     <div class='protein-info-wrapper'>
                                         <p class='protein-info'>Gene: {{ extendedData.gene_names }}</p>
-                                        <p class='protein-info protein-subtitle'>Protein Names: {{
-                                            extendedData.protein_name }}
+                                        <p class='protein-info'>Protein Names: {{
```
Comment (Yongfang-Qin) [inline fatplant/src/app/components/pages/datapages/protein-details-new/protein-details-new.component.html]: why remove the protein-subtitle from the list?

Comment (Yongfang-Qin) [reply fatplant/src/app/components/pages/datapages/protein-details-new/protein-details-new.component.html]: why replace `extendedData.protein_name` with ` headingName`

File: fatplant/src/app/components/pages/datapages/protein-details-new/protein-details-new.component.html [hunk 3]
```diff
@@ -52,14 +55,14 @@ <h2 *ngIf="homologs" class='title'>Homologs</h2>
...
+                                            headingName }}
                                         </p>
                                         <p class='protein-info'>RefSeq ID: {{ extendedData.refseq_id }}</p>
                                         <p class='protein-info' *ngIf="extendedData.tair_id">Tair ID: {{
                                             extendedData.tair_id }}
                                         </p>
```
Comment (Yongfang-Qin) [inline fatplant/src/app/components/pages/datapages/protein-details-new/protein-details-new.component.html]: same as above, why remove `protein-subtitle`

File: fatplant/src/app/components/pages/datapages/protein-details-new/protein-details-new.component.html [hunk 4]
```diff
@@ -90,7 +93,7 @@ <h3 class='protein-subtitle'>Protein Function</h3>
                                     <p class='protein-function' *ngIf='extendedData.function.length === 0'>No
                                         function specified for this entry.</p>
                                     <b class='protein-subtitle'>Keywords:</b>
-                                    <p>
+                                    <p class= 'protein-function'>
```
Comment (Yongfang-Qin) [inline fatplant/src/app/components/pages/datapages/protein-details-new/protein-details-new.component.html:96]: we need to add margin for the whole box of `Protein Function`. it isn't good if we add margin to the element of the box

File: fatplant/src/app/components/pages/datapages/protein-details-new/protein-details-new.component.scss (hunk 1 above)
Comment (Yongfang-Qin) [reply fatplant/src/app/components/pages/datapages/protein-details-new/protein-details-new.component.scss:98]: do we need to make change to ` no-data-message` style?

File: fatplant/src/app/components/pages/datapages/protein-details-new/protein-details-new.component.html (hunk 2 above)
Comment (Yongfang-Qin) [reply fatplant/src/app/components/pages/datapages/protein-details-new/protein-details-new.component.html]: `extendedData.protein_name ` doesn't show correctly, replace it with `headingName`

Comment (harshavardhanchintapatla) [reply fatplant/src/app/components/pages/datapages/protein-details-new/protein-details-new.component.html]: Regarding protein-subtitle, Protein-subtitle class has color attribute so preotein_name and Uniport Id will be highlighted in green in Identification section. 
I checked it with previous page and I will add it back those classes and commit the code.

File: fatplant/src/app/components/pages/datapages/protein-details-new/protein-details-new.component.html (hunk 4 above)
Comment (harshavardhanchintapatla) [reply fatplant/src/app/components/pages/datapages/protein-details-new/protein-details-new.component.html:96]: Screenshot 2024-04-19 105221
Regarding adding protein-function class to the keywords section, protein function class has just spacing attribute and as shown in the screenshots only the keywords section doesn't have any class to specify spacing so like other sections i added the same class for this too

File: fatplant/src/app/components/pages/datapages/protein-details-new/protein-details-new.component.scss (hunk 1 above)
Comment (harshavardhanchintapatla) [reply fatplant/src/app/components/pages/datapages/protein-details-new/protein-details-new.component.scss:98]: No we are not changing any attributes for no-data-message class, I have added spacing for the .function-header class  from line no.94

File: fatplant/src/app/components/pages/datapages/protein-details-new/protein-details-new.component.html (hunk 2 above)
Comment (Yongfang-Qin) [reply fatplant/src/app/components/pages/datapages/protein-details-new/protein-details-new.component.html]: please add it back. We need to confirm why they add it before we remove it

File: fatplant/src/app/components/pages/datapages/protein-details-new/protein-details-new.component.html (hunk 3 above)
Comment (Yongfang-Qin) [reply fatplant/src/app/components/pages/datapages/protein-details-new/protein-details-new.component.html]: Add it back

File: fatplant/src/app/components/pages/datapages/protein-details-new/protein-details-new.component.html (hunk 4 above)
Comment (Yongfang-Qin) [reply fatplant/src/app/components/pages/datapages/protein-details-new/protein-details-new.component.html:96]: Got your point, it's ok to add it

File: fatplant/src/app/components/pages/datapages/protein-details-new/protein-details-new.component.scss (hunk 1 above)
Comment (Yongfang-Qin) [reply fatplant/src/app/components/pages/datapages/protein-details-new/protein-details-new.component.scss:98]: Weird, it shows that we need a new line

**Review by Yongfang-Qin — APPROVED** [review]

**Pull request merged**